enclave_manager/avalon_enclave_manager/enclave_manager.py

```diff
@@ -473,33 +497,70 @@ def start_enclave_manager(config):
                      "exiting SGX Enclave manager: {err}")
         exit(1)
 
-    try:
-        sleep_interval = int(config["EnclaveManager"]["sleep_interval"])
-    except Exception as err:
-        logger.error("Failed to get sleep interval from config file. " +
-                     "Setting sleep interval to 10 seconds: %s", str(err))
-        sleep_interval = 10
+    if int(config["WorkloadExecution"]["sync_workload_execution"]) == 1:
+        # Binding with ZMQ Port
+        try:
+            socket = bind_zmq_socket(config.get("EnclaveManager")["zmq_port"])
+            logger.info("ZMQ Port hosted by Enclave")
+        except Exception:
+            logger.exception("Failed to bind socket")
```

Comment:
Do you want to retry socket binding in case of exception or just halt enclave manager process?
If you want to halt, you may need to add exit() in the except block.

Reply:
Thanks for pointing out this error. I have updated in the new commit.

Reply:
You don't need to raise separate commits to address review comments. You should update the relevant commit after all the rework using "git commit --amend"

Reply:
Okay, Will use the same if there are further changes.